packages/guui/index.js

```diff
@@ -7,7 +7,7 @@ import { extractCritical } from 'emotion-server';
 import React, { Component } from 'react';
 import { hydrate as hydrateApp, render } from 'react-dom';
 import { renderToString as reactRenderToString } from 'react-dom/server';
-import styled from 'react-emotion';
+import styled, { css } from 'react-emotion';
```

Comment:
I know the plan is to use these directly but I've hacked this here (and in my other PRs) for expediency just so I could test stuff.